Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierWebsiteQuestion.cshtml.cs

```diff
@@ -14,11 +15,11 @@ namespace CO.CDP.OrganisationApp.Pages.Supplier;
 public class SupplierWebsiteQuestionModel(IOrganisationClient organisationClient) : PageModel
 {
     [BindProperty]
-    [Required(ErrorMessage = "Select an option")]
+    [Required(ErrorMessage = nameof(StaticTextResource.Supplier_WebsiteQuestion_ErrorSelectOption))]
```

Comment:
Use following pattern for Required attribute

https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/main/docs/development/cookbooks/02-how-to-localize-ui-text.adoc#how-to-localize-form-validation-errors